<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountantsPage</class>
 <widget class="QWidget" name="AccountantsPage">
  <property name="windowTitle">
   <string>Gestion des Comptables</string>
  </property>
  <layout class="QVBoxLayout" name="mainVerticalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>

   <item>
    <layout class="QHBoxLayout" name="headerLayout">
     <item>
      <widget class="QLabel" name="label_titre">
       <property name="text">
        <string>GESTION DES COMPTABLES</string>
       </property>
      </widget>
     </item>
     
     <item>
      <spacer name="headerSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>

     <item>
    <widget class="QLineEdit" name="input_search">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>üîç Rechercher par nom, courriel ou r√¥le...</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>

     <item>
      <widget class="QPushButton" name="btn_ajouter">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>+ Ajouter un comptable</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>

   <item>
    <widget class="QTableWidget" name="table_accountants">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
</ui>